drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -533,7 +533,13 @@ def can_see_client_details?(user)
               dobs = scope.pluck(:client_id, c_t[:DOB]).to_h
               scope.pluck(:client_id, p_t[:id], :first_date_in_program, :move_in_date).
                 each do |client_id, project_id, entry_date, move_in|
-                  days = (move_in - entry_date).to_i
+                  days = if move_in < entry_date
```

Comment:
A idle thought... Would `days = [ (move_in - entry_date).to_i, 0 ].max` be a good pattern, or not?

Reply:
Yeah, that'd probably work as well.  I was tempted to use clamp as well.